pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -71,7 +68,7 @@ type TokensControllerOptions struct {
 }
 
 // NewTokensController returns a new *TokensController.
-func NewTokensController(cl clientset.Interface, options TokensControllerOptions) *TokensController {
+func NewTokensController(serviceAccounts, secrets cache.SharedIndexInformer, cl clientset.Interface, options TokensControllerOptions) *TokensController {
```

Comment:
Type the informers please.  We have generated types for SecretInformer and ServiceAccoutnInfomer